CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
 - Support loading Firebase Functions packaged as an ES module. (#3485)
 - Fixes Cloud Storage emulator missing classes. (#3541)
 - Add missing properties to Authentication functions `context` in the Functions emulator. (#3536)
+- Firestore Emulator UI now supports deleting documents and collections recursively.
+- Fixes some Storage Emulator UI errors.
```

Comment:
Any links to issues for "some errors"?

Reply:
Issues are on a different repo, such as https://github.com/firebase/firebase-tools-ui/issues/576. I don't know the right convention for that.

Reply:
I think just linking them like that works, GitHub makes it pretty nice and short. But not a blocker.

Reply:
Just tried and the nice and short formatting doesn't work in regular Markdown files. Just comments.